omnigibson/robots/behavior_robot.py

```diff
@@ -363,8 +361,8 @@ def base_footprint_link(self):
         """
         return self._links[self.base_footprint_link_name]
 
-    def get_position_orientation(self):
-        return self.base_footprint_link.get_position_orientation()
+    def get_position_orientation(self, clone=True):
```

Comment:
Lets undo clones here too. It's a tricky interface imo.

Reply:
resolved in person -- keep for now, revert later